iFixit/wiki-table-editor
# Firefox: Fix issue with selecting text within inputs in table rows

This pull makes it so firefox users can highlight text within inputs
that are present in table rows. To do this we copied a draggable-row
file from reactabular-dnd and made some modifications that:

1. Make it so connectDragSource is not called on the entire row but on
   each individual cell.
2. Explicitly sets the entire row to be the source of our dragPreview so
   when we drag a row by any cell the preview a user sees isn't of just
   that cell but of the entire row.

We use this draggableRow in our wiki-table-editor.jsx and add a new
DnDCell class that will only connect the drag source to a cell if it
has the canDragCell prop that is only present on our row drag handle. Otherwise a
standard table cell is returned.

QA
------------------------
- Verify the table editor here: https://me.cominor.com/Item/Edit/T5_Torx_Screwdriver still works correctly in all browsers.
- Verify you can highlight the text within the input in all browsers.

Huge thanks to @danielj41 for helping us figure out this issue!
